src/components/sidebar/routeListView/RouteItem.tsx

```diff
@@ -1,38 +1,40 @@
-import classNames from 'classnames';
 import { inject, observer } from 'mobx-react';
-import Moment from 'moment';
 import React from 'react';
-import { FiInfo } from 'react-icons/fi';
-import { Button } from '~/components/controls';
-import { IRoute, IRoutePath } from '~/models';
-import navigator from '~/routing/navigator';
-import QueryParams from '~/routing/queryParams';
-import routeBuilder from '~/routing/routeBuilder';
-import subSites from '~/routing/subSites';
-import { MapStore } from '~/stores/mapStore';
+import { ContentItem, ContentList, Tab, Tabs, TabList } from '~/components/shared/Tabs';
+import { IRoute } from '~/models';
 import { RouteListStore } from '~/stores/routeListStore';
-import LineHelper from '~/util/LineHelper';
-import TransitTypeHelper from '~/util/TransitTypeHelper';
-import { toDateString } from '~/util/dateHelpers';
-import ToggleSwitch from '../../controls/ToggleSwitch';
-import SidebarHeader from '../SidebarHeader';
-import * as s from './routeItem.scss';
+import RoutePathListTab from './RoutePathListTab';
+import RouteTab from './RouteTab';
 
 interface IRouteItemProps {
-    routeListStore?: RouteListStore;
-    mapStore?: MapStore;
     route: IRoute;
+    isEditingDisabled: boolean;
+    routeListStore?: RouteListStore;
+}
+
+interface IRouteItemState {
+    selectedTabIndex: number;
+    areAllRoutePathsVisible: boolean;
 }
 
 @inject('routeListStore', 'mapStore')
 @observer
-class RouteItem extends React.Component<IRouteItemProps> {
+class RouteItem extends React.Component<IRouteItemProps, IRouteItemState> {
+    constructor(props: any) {
+        super(props);
+        this.state = {
+            selectedTabIndex: 1,
```

Comment:
Should this be set to 0 as default? If not then maybe "reitinsuunnat" tab should be the "main tab" and be on the right rather than "reitin tiedot"